cernopendata/templates/cernopendata_records_ui/records/detail.html

```diff
@@ -0,0 +1,94 @@
+{%- extends "invenio_records_ui/detail.html" %}
+
+{%- block page_body %}
+<div class="container">
+  <div class="panel panel-default">
+    <div class="panel-heading">
+      <h2 style="display: inline;">
+        {{record.title}}
+      </h2>
+      <div class="row">
+        <div class="col-md-12">
+          {% for author in record.authors %}
+          <small>{{author.name}}</small>
+          {% endfor %}
+          <span class="label label-success">{{record.collaboration}}</span>
+          <span class="label label-success">{{record.year}} Run</span>
+        </div>
+      </div>
+      <p></p>
+      <div class="row">
+        <div class="col-md-12">
+          <label>Cite as:</label>
+          {% for author in record.authors %}
+            <span>{{author.name}}.</span>
+          {% endfor %}
+          <span>{{record.collaboration}} ({{record.year}}). {{record.title}}. CERN Open Data Portal.
+            {% for id in record.identifiers %}
+              <label>{{id.type}}:</label>
+              <a href="http://doi.org/{{id.id}}">{{id.id}}</a>
+            {% endfor %}
+          </span>
+        </div>
+      </div>
+      <p></p>
+      <div class="row">
+        <div class="col-md-12">
+          <label>Experiment:</label>
+          <span><a href="/search?p=CERN+LHC">{{record.experiment}}</a></span>
```

Comment:
Note that `search?p=...` is now `search?q=...` so these links wouldn't work... Please change. 

(Although we may perhaps want to implement the former to keep compatibility... let's muse in another issue.)

Reply:
True didn't notice, will update!